docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -38,7 +38,7 @@ To use Iceberg, you need:
 
 * Network access from the Trino coordinator and workers to the distributed
   object storage.
-* Access to a Hive metastore service (HMS), AWS Glue or a `Nessie server
+* Access to a Hive metastore service (HMS), AWS Glue, or a `Nessie server
```

Comment:
List needs to be extended.. 

Hive metastore service (HMS), a AWS Glue catalog, a JDBC catalog, a REST catalog, or a Nessie server.

And dont link to Nessie .. if anything we could link to the later sections about these in the doc.

Reply:
Unlinked Nessie. @m57lyra was Nessie removed entirely? When I search the past few versions of Trino docs, I don't get any hits. I had fetched upstream and so forth before creating this PR.

Reply:
Nessie should stay and is brand new. @m57lyra will abandon her current PR and restart with a few new ideas and this work.

Reply:
Linked to all sections as Manfred suggested.